css/index.css

```diff
@@ -71,20 +71,42 @@ p {
     margin: 0 auto;
 }
 
+.header{
+    font-size:14px;
+    width: 800px;
+    height:80px;
+    margin-top: 2%;
+    display:flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+}
+.logo{
+    margin-bottom: 3%;
+}
+.navigation{
+    width:500px;
+    height:20px;
+    display:flex;
+    justify-content:space-between;
+}
+
 .top-content {
     display: flex;
     flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
+    justify-content: space-between;
+    margin:20px 0px;
     border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
     width: 48%;
     padding: 0 1%;
-    padding-bottom: 20px;  
+    padding-bottom: 20px;
```

Comment:
On line 105 you have padding: 0 1%, which is setting left and right padding to 0, and top and bottom padding to 1%. Then here on 106 you are overriding bottom padding to 20px. I would remove both and instead use 
padding: 1% 0 20px
https://www.w3schools.com/css/css_padding.asp